### ISSUE TITLE
Not using any context

### ISSUE THREAD

**Issue by oderwat** [nats-io/natscli#637]
While testing, I ran into the situation that I had a context active that defined a creds file, so I gave a different server (web socket) and the test succeeded because of the "invisible" creds file. I did not realize that at first. Then I wondered how to get rid of "any" context and just get what is on the command line. Giving an empty context or creds file as parameter did not succeed. So, I created a "none" context from "no parameters" which kind of works but still sets the 127.0.0.1 server. It seems that `rm ~/.config/nats/context.txt` is the only way to get rid of a selected context and return to nats without any context selection. I think that nats-cli should have something like `nats context none` to make this a documented feature.

**Comment by oderwat** [comment]
> hmm, I thought I fixed the thing where you cant remove the selected context but maybe only when its the only remaining one.

It is not about deleting contexts (I have quite some context I do not want to lose).

> Will think about it, maybe add a --no-context flag that just disable context loading entirely.

I just want to "reset" the context, and go to the state where you do not have any context selected to be sure that nothing is influencing my commands. 

Having a --no-context would work too, but removing "any" context probably should still be an option?

**Comment by ripienaar** [comment]
So essentially you want to just unselect the context? or operated without a selected context which is what `--no-context` would do?

**Comment by oderwat** [comment]
I was thinking about "unselecting" but just skipping it for a command seems to be also a useful variant. Especially in scripts because you make no change to the context of the user and need to restore it after the script is done.

**Comment by ripienaar** [comment]
Yeah, just being able to opt out of using the selected context - so using no context - seems like the least disruptive mode to me and a way to make scripts safe.  Though personally I use contexts in scripts but then I supply their full path and not a name or leaving it up to the selected one.

**Comment by ripienaar** [comment]
https://github.com/nats-io/natscli/pull/641
